pkg/wallet/wallet.go

```diff
@@ -194,10 +170,8 @@ func (w *Wallet) JSON() ([]byte, error) {
 }
 
 // Close closes the internal rw if its an io.ReadCloser.
+// Deprecated: will be removed in future versions.
```

Comment:
Not sure about this. We can and should scrub private key memory here.

Reply:
@fyrchik? It might get deprecated, but I don't think it is now.

Reply:
Fixed!

Reply:
Nothing changed. Forgot to push?

Reply:
Yes, fixed.